junit-platform-suite-api/src/main/java/org/junit/platform/suite/api/SelectMethod.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.suite.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+
+public @interface SelectMethod {
+
+	/**
+	 * The name of the method to select.
+	 */
+	String name();
+
+	/**
+	 * The parameter types of the method to select, in the form "int,String".
+	 */
+	String parameterTypes() default "";
```

Comment:
As an after thought. Since this is going to be a list of parameters, perhaps we can use  `Class<?>[]` and use `ClassUtils.nullSafeToString` to create the `methodParameterTypes` for the `MethodSelector`. One of the `MethodSelector` constructors does the same thing.

Reply:
I've only found two methodSelector constructors that take methodParameterTypes as input, both take it as `string`, not as `Class<?>[]`. Am I supposed to convert `Class<?>[]` to `String` myself with `ClassUtils.nullSafeToString`?

Reply:
For now I think we can use `nullSafeToString`. Once everything works properly and is tested we can look at adding a new overloaded method to `MethodSelector`.